src/docs/content/observability/observability-example/setup.md

```diff
@@ -0,0 +1,230 @@
+---
+title: "Setup"
+date: 2020-03-10T13:29:23-06:00
+draft: false
+weight: 3
+tags:
+  - OCI Logging
+  - OCI Notifications
+  - OCI Service Connector Hub
+  - Deploy MuShop
+  - fluentd
+  - logging
+  - alarms
+  - notifications
+---
+
+### Deploy MuShop
```

Comment:
Where is deploying MuShop? If only on OKE, a quick note pointing that should be included, as MuShop can be deployed on different places including other clouds.

Reply:
Added a note while deploying MuShop.